packages/apps/reputation-oracle/server/src/modules/kyc/kyc.service.spec.ts

```diff
@@ -218,46 +186,73 @@ describe('Kyc Service', () => {
     const mockKycUpdate = {
```

Comment:
```suggestion
    mockKycUpdate: UpdateKycStatusDto
```
instead of `} as UpdateKycStatusDto;`